examples/fullstack/frontend/README.md

```diff
@@ -6,8 +6,18 @@ demo.
 > NOTE: The desktop app currently has only been tested on macOS. Testing on
 > Linux or Windows and help with updating docs is appreciated!
 
+## Requirements
+
+- [Flutter](https://flutter.dev/docs/get-started/install)
+
 ## Environment
 
+Before beginning, enable macos development:
+
+```shell
+flutter config --enable-macos-desktop
```

Comment:
I suggest this (from [examples/fullstack/README.md](https://github.com/GoogleCloudPlatform/functions-framework-dart/tree/main/examples/fullstack#build-and-run-the-frontend)):

If you have not enabled desktop support for your Flutter installation, do so now with one of the following commands, as appropriate for your system:

```
flutter config --enable-windows-desktop
flutter config --enable-macos-desktop
flutter config --enable-linux-desktop
```

Reply:
Fix applied!